Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -666,6 +668,69 @@ extension HomepageViewController: HomepageContextMenuHelperDelegate {
     }
 }
 
+// MARK: - Status Bar Background
+private extension HomepageViewController {
+
+    var statusBarView: UIView? {
+        guard let statusBarFrame = statusBarFrame else {
+            return nil
+        }
+
+        // We put a view behind the status bar to be able to give it a background color.
+        // We use a tag to be able to retrieve the view instead of recreating it every time
+        // the user scrolls.
+        let tag = 11111
```

Comment:
@thatswinnie I'm wondering if it would be good to consolidate tag number in general under a struct or enum or something else? What do you think? 

`11111` is a random number and someone else might use the same tag number

Reply:
Actually, instead of a tag could you try `ObjectIdentifier` and maybe something like `print(ObjectIdentifier(view))` will give you a value that you can use in a variable. Try it I am just suggesting it to avoid having multiple views with the same tag.